hejrun.py

```diff
@@ -18,19 +18,13 @@
 
 #### Override print with custom version that always flushes to stdout so we have up-to-date logs
 def print_flush(string):
-    print string
+    print(string)
```

Comment:
We are still running Python 2.X, print() is python 3. The comment about 2.4 can go, but its not probably 2.6.

Reply:
It is better to leave the brackets because if `print` is a function they are needed, if `print` is a statement everything between brackets will be understood as the argument of the statement, so they never hurt.

Reply:
I should add, this is true in this particular case, the reason is because

`string` and `(string)` are exactly the same thing for python

but

`(string,)` instead is a tuple.

Reply:
Oh sorry I should have emphasised. I don't have a problem with the print itself, but we need to import it. This comment was more a reminder for me.

Reply:
Sorry Marian, not sure I follow.  I don't think we need to import it?

Reply:
It is not needed. But I suppose it doesn't hurt either. I'll merge.